org.bitbucket.mystery_keeper.VapourSynthEditor.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: org.bitbucket.mystery_keeper.VapourSynthEditor
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: vsedit
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-videos
+add-extensions:
+  org.bitbucket.mystery_keeper.VapourSynthEditor.Plugin:
+    version: '1'
+    directory: lib/vapoursynth
+    subdirectories: false
+    autodelete: true
+cleanup:
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /lib/pkgconfig
+modules:
+  - com.vapoursynth.VapourSynth.yaml
+
+  - name: vapoursyntheditor
```

Comment:
Looks like it's duplicated? Also, if it's supposed to work only on x86_64, you also need a flathub.json file to limit the build to one arch.

Reply:
It's kind of duplicated, but with one difference. The issue was that qmake added the suffix `-32-bit` to all builds other than `x86_64`, so I handle those separately.

Reply:
`skip-arches` vs `only-arches` in the second, never mind. You should be able use bash syntax to check if file exists, this would help with deduplication.